jwt.js

```diff
@@ -209,7 +209,12 @@ function fastifyJwt (fastify, options, next) {
         secretCallbackVerify(request, decodedToken, callback)
       },
       function verify (secretOrPublicKey, callback) {
-        jwt.verify(token, secretOrPublicKey, options, callback)
+        jwt.verify(token, secretOrPublicKey, options, (err, result) => {
+          if (err && err instanceof jwt.TokenExpiredError) {
```

Comment:
You don't need to add an existence check of `err` :)
```suggestion
          if (err instanceof jwt.TokenExpiredError) {
```